components/ProfileImageInfo.tsx

```diff
@@ -1,45 +1,163 @@
-import React from 'react'
+import React, { useState } from 'react'
 import UserInfoImage from './UserInfoImage'
-import { UserInfo } from '../@types/user'
 import Image from 'next/image'
 import Link from 'next/link'
 import styles from '../scss/profileImageInfo.module.scss'
+import { UserInfo } from '../@types/user'
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  ModalTitle
+} from 'react-bootstrap'
+import { Button } from './theme/Button'
+import { signIn } from 'next-auth/react'
+import { Button as BsButton } from 'react-bootstrap'
+import { useMutation } from '@apollo/client'
+import { UNLINK_DISCORD } from '../graphql/queries/unlinkDiscord'
+import { useRouter } from 'next/router'
+
 type ProfileImageInfoProps = {
   user: UserInfo
 }
 
 const ProfileImageInfo: React.FC<ProfileImageInfoProps> = ({ user }) => {
+  const [unlinkDiscord] = useMutation(UNLINK_DISCORD)
+  const router = useRouter()
+
+  const [show, setShow] = useState(false)
+
+  const handleClose = () => setShow(false)
+  const handleShow = () => setShow(true)
+
+  const handleConnectClick = () => {
+    signIn('discord', { callbackUrl: '/discord/success' })
+    handleShow()
+  }
+
+  const handleUnlinkClick = async () => {
+    try {
+      handleClose()
+      await unlinkDiscord()
+      router.reload()
+    } catch (e) {
+      // Calling this function make DiscordBar in storyshots fail with:
+      // Warning: An update to DiscordBar inside a test was not wrapped in act(...)
+      // Track it on https://github.com/garageScript/c0d3-app/issues/2257
+      // Sentry.captureException(e)
+    }
+  }
+
   return (
     <div className="card shadow-sm">
       <div
-        className={`ms-auto me-auto mt-4 ${styles['profile_image_container']}`}
+        className={`ms-auto me-auto mt-4 mb-3 ${styles['profile_image_container']}`}
       >
         <UserInfoImage user={user} className="mt-4" />
       </div>
-      <h2 className="text-center mt-4">
+      <h4 className="text-center mt-4">
         {`${user.firstName} ${user.lastName}`}
-      </h2>
+      </h4>
       <div className="d-flex justify-content-center">
-        <h4 className="text-muted">{'@' + user.username}</h4>
+        <h5 className="text-muted">{`@${user.username}`}</h5>
```

Comment:
Note: h1/2/3/4/5/6 tags are usually used to help structure the page so screen readers identify headings of sections.
They shouldn't be used to comply with visual requirements (see: https://www.htmhell.dev/27-a6/).
You can use Bootstrap's className="fs-5" instead which should be the same visually as using h5 (see: https://getbootstrap.com/docs/5.0/utilities/text/#font-size).

You don't need to change this now since it was already like this, but this is something to aware of in the future.